packages/openapi-generator/src/wrapper-files/readme.ts

```diff
@@ -0,0 +1,39 @@
+import { codeBlock } from '@sap-cloud-sdk/util';
+import { OpenApiDocument } from '../openapi-types';
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Generate the readme for an openapi client.
+ * @param openApiDocument Parsed service.
+ * @returns The readme contents.
+ */
+export function readme(openApiDocument: OpenApiDocument): string {
+  return codeBlock`# ${openApiDocument.npmPackageName}
```

Comment:
Do you need the codeBlock here? It mostly takes care of indentation, so might be useful, just doesn't seem that this is used here.